### PR TITLE
ROX-13189: using real data for namespace network policies

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR uses real data for the network policies section of the deployment and namespace side panels

NOTE: This will always show "no network policies" because we need to wait on [ROX-13433](https://issues.redhat.com/browse/ROX-13433) to see the `policyIds` actually populated with network policy ids. You can test it out by passing hardcoded network policy ids

## Checklist
- [ ] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Screenshot 2022-11-14 at 11 48 25 AM
Screenshot 2022-11-14 at 11 48 38 AM
Screenshot 2022-11-14 at 12 37 58 PM

### CODE REVIEW

**Review by vjwilson — APPROVED** [review]
/lgtm

File: ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ const TopologyComponent = ({ model }: TopologyComponentProps) => {
 
     function onSelect(ids: string[]) {
         const newSelectedId = ids?.[0] || '';
-        const newSelectedEntity = getNodeById(model, newSelectedId);
+        const newSelectedEntity = getNodeById(model?.nodes, newSelectedId);
```
Comment (alwayshooin) [inline ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx:108]: shouldn't it just not call the function if there are no nodes?

**Review by alwayshooin — APPROVED** [review]
lgtm! just one comment in the code

File: ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx:108]: It's more for typescript. Seems like `nodes` is actually optional

**Pull request merged**